schemas/context/opportunity-crm.schema.json

```diff
@@ -0,0 +1,141 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/opportunity-crm",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "CRM Opportunity",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/opportunity"
+  ],
+  "description": "CRM specific opportunity fields.",
+  "definitions": {
+    "opportunity-crm": {
+      "properties": {
+        "xdm:name": {
```

Comment:
As stated earlier, the context has to be represented as an object, so that a meaningful representation of properties can be established if not absolutely stated.